<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Log</class>
 <widget class="QWidget" name="Log">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/File/images/coffee.png</normaloff>:/File/images/coffee.png</iconset>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="next_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_all_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清除历史</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
